prisma/schema.prisma

```diff
@@ -82,4 +82,23 @@ model Target {
   deleted_at  DateTime?
 
   @@map("targets")
+}
+
+model Report_Target {
+  id          Int           @id @default(autoincrement())
+  user_id     Int
+  target_id   Int
```

Comment:
https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-many-relations